confmap/converter.go

```diff
@@ -35,4 +35,6 @@ func NewConverterFactory(f CreateConverterFunc) ConverterFactory {
 type Converter interface {
 	// Convert applies the conversion logic to the given "conf".
 	Convert(ctx context.Context, conf *Conf) error
+	// Type returns the type of the converter.
+	Type() string
```

Comment:
Adding a method to a non-sealed interface is a breaking change, we can't do it this way, since `confmap` is 1.0. As it stands, converters don't have a type